pkg/routes/v1_test.go

```diff
@@ -1,103 +1,95 @@
 package routes
```

Comment:
Why do we have to comment out all of them?

Reply:
because the testcase cannot run successful. The keys on `expectedRoutes` are not unique. Examples:
```
"/api/v1/projects": {
	Method:  "GET",
	Handler: "github.com/dwarvesf/fortress-api/pkg/handler/project.IHandler.List-fm",
},
"/api/v1/projects": {
	Method:  "POST",
	Handler: "github.com/dwarvesf/fortress-api/pkg/handler/project.IHandler.Create-fm",
},
````

Reply:
if I doesn't add api create project, the testcase will run fail

Reply:
nvm, i will fix it

Reply:
updated